evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -13,15 +13,15 @@ <h4 class="{% if not forloop.first %} mt-3 {% elif contributor %} me-auto {% end
                 >
                     {% blocktranslate %}I can't give feedback about this contributor{% endblocktranslate %}
                 </button>
-                <confirmation-modal class="mark-no-answer-modal" confirm-button-class="btn-danger" id="mark-no-answer-modal">
-                    <span slot="title">{% translate 'Bewertungen für Person zurücknehmen' %}</span>
-                    <span slot="action-text">{% translate 'Antworten zurücknehmen' %}</span>
+                <confirmation-modal class="mark-no-answer-modal" confirm-button-class="btn-danger">
+                    <span slot="title">{% translate 'Take back answer for this person' %}</span>
```

Comment:
`answers`

Reply:
Should we use more descriptive/precise wording than "Take back"? I can see how someone could understand "Take back" to mean: "Don't include with my submission, but keep the data, so I could later decide to include it again if I want to". We may want to use "reset" or "delete" or any other established terms for destructive operations.

Reply:
I suggest using `Delete my answers about this person` then.